os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1010,6 +1011,7 @@ u32 app_mpu_nocache_init(void)
 	mpu_cfg.attr_idx = MPU_MEM_ATTR_IDX_WB_T_RWA;
 	mpu_region_cfg(mpu_entry, &mpu_cfg);
 
+	g_mpu_last_entry = mpu_entry + 1;
```

Comment:
This is not the last entry. It's (last entry + 1). Let's name it as `g_mpu_nregion_allocated`